src/fuel-flow-to-load-test-workspace/fuel-flow-to-load-test-workspace.service.ts

```diff
@@ -108,7 +116,7 @@ export class FuelFlowToLoadTestWorkspaceService {
 
     entity.testBasisCode = payload.testBasisCode;
     entity.averageDifference = payload.averageDifference;
-    entity.numberOfHoursUsed = payload.numberOfHoursExcludedCofiring;
+    entity.numberOfHoursUsed = payload.numberOfHoursUsed;
```

Comment:
I agree that this was probably a mistake and should be corrected, but I think we may want to include `numberOfHoursExcludedCofiring`, too:
```suggestion
    entity.numberOfHoursUsed = payload.numberOfHoursUsed;
    entity.numberOfHoursExcludedCofiring = payload.numberOfHoursExcludedCofiring;
```